test/addons_test.go

```diff
@@ -24,8 +24,10 @@ import (
 	"github.com/mesosphere/kubeaddons/pkg/test/cluster/kind"
 )
 
-const defaultKubernetesVersion = "1.15.6"
-
+const (
+	defaultKubernetesVersion = "1.15.6"
+	patchStorageClass = "{\"metadata\": {\"annotations\":{\"storageclass.kubernetes.io/is-default-class\":\"false\"}}}"
```

Comment:
Would this be better/cleaner?
```suggestion
	patchStorageClass = `{"metadata":{"annotations":{"storageclass.kubernetes.io/is-default-class":"false"}}}`
```

Reply:
Thought i tried that but with json issues. Ill retry, maybe removing the first quotes will work.